content/documentation/pages/6-recipes/1-polyglot/2-polyglot-python-task.md

```diff
@@ -6,31 +6,62 @@ description: 'Python/Docker as SCDF Task'
+This recipe shows how to run a custom Python script as a [Data Flow Task](http://docs.spring.io/spring-cloud-dataflow/docs/%scdf-version-latest%/reference/htmlsingle/#spring-cloud-dataflow-task) and how to orchestrate later as [Composed Tasks](http://docs.spring.io/spring-cloud-dataflow/docs/%scdf-version-latest%/reference/htmlsingle/#spring-cloud-dataflow-composed-tasks).
 
-**TODO: Describe what the Task is doing**
+The approach requires the scripts to be bundled as docker images and is applicable for the `Local` and `Kubernetes` runtime platform.
 
-A small helper class [TaskStatus](https://github.com/tzolov/scdf-polyglot-experiments/blob/master/python_task_with_status/util/task_status.py) is used to write task state to the Task table created by Data Flow.
+Following diagram helps to visualize the component involved and the control flow of this approach.
 
-The source code can be found [here](https://github.com/tzolov/scdf-polyglot-experiments/tree/master/python_task_with_status)
+![SCDF Python Tasks](images/python_docker_task_self_managed_status.png)
 
-**TODO provide a .zip file of all the source code**
-**TODO: Source code to be moved into data flow samples repo in a directory structure that mimics the dataflow.io directory structure. There is already some content in there to get an idea of the directory structure.**
+When launched, the custom script executes its task and completes with either a successfully or failure status.
+Because this is not a standard Sprint Cloud Task, the script is responsible to manage its life cycle state within the Data Flow database!
+Utilities are provided to help handle the launch arguments and manage the task status within the Data Flow database.
 
-## Description
+<!--TIP-->
 
-The [python_task.py](https://github.com/tzolov/scdf-polyglot-experiments/blob/master/python_task_with_status/python_task.py) script will sleep for 60 seconds and exit successfully unless the command line argument `error.message` is passed to the application.
+The source code can be found in the samples GitHub [repo](https://github.com/spring-cloud/spring-cloud-dataflow-samples/tree/master/dataflow-website/recipes/polyglot/polyglot-python-task) or downloaded as a zipped archive: [polyglot-python-task.zip](https://github.com/spring-cloud/spring-cloud-dataflow-samples/raw/master/dataflow-website/recipes/polyglot/polyglot-python-task.zip). Follow the [Build](#build) instructions for building and using the project.
+
+<!--END_TIP-->
+
+## Quick Start
+
+Follow the [installation instructions](/documentation/master/installation/kubernetes/) to set up Data Flow on Kubernetes. Launch the Data Flow server with the task feature enabled!
+
+Register and launch a python scrip as SCDF task:
+
+```bash
+dataflow:>app register --type task  --name python-task-with-status --uri docker://tzolov/python_task_with_status:0.1
```

Comment:
We are shipping sample apps to io.spring repo and as well to Docker Hub. We should switch to do the same here as well.

Reply:
Image is uploaded to `springcloud/python-task-with-status:0.1` DH